.github/workflows/pytest.yaml

```diff
@@ -1,5 +1,5 @@
 name: tests
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Just as a note, I was in doubt whether the action would only be triggered when opening a PR and I learnt that PRs are triggered on three subactivities by default: opened, synchronize, and reopened. So all subsequent commits to the PR will be checked ✅